tesseract_collision/bullet/CMakeLists.txt

```diff
@@ -27,6 +27,14 @@ else()
   set(BULLET_LIBRARY_DIRS_ABS ${BULLET_LIBRARY_DIRS})
 endif()
 
+set(BULLET_LIBRARIES_ABS "")
```

Comment:
Does the code above not produce absolute paths?

Reply:
No, that just changes the include directory. The libraries themselves are still without a path, and that was causing a lot of problems because it was finding them in the wrong place. Without the current changes, it wasn't even searching the directory specified by the bullet path because there was no `link_directories()` call.

Reply:
This is the content of `BulletConfig.cmake`

```cmake
set ( BULLET_FOUND 1 )
set ( BULLET_USE_FILE     "lib/cmake/bullet/UseBullet.cmake" )
set ( BULLET_DEFINITIONS  "-DBT_USE_DOUBLE_PRECISION" )
set ( BULLET_INCLUDE_DIR  "include/bullet" )
set ( BULLET_INCLUDE_DIRS "include/bullet" )
set ( BULLET_LIBRARIES    "LinearMath;Bullet3Common;BulletInverseDynamics;BulletCollision;BulletDynamics;BulletSoftBody" )
set ( BULLET_LIBRARY_DIRS "lib" )
set ( BULLET_ROOT_DIR     "C:/ws/tesseract_noetic_ci/tesseract_ws/install_isolated" )
set ( BULLET_VERSION_STRING "3.25" )
```

`BULLET_LIBRARIES` does not include a path, so it basically searches the filesystem.